DNN Platform/DotNetNuke.Web.Client/ClientResourceManager.cs

```diff
@@ -305,7 +312,21 @@ public static void RegisterScript(Page page, string filePath, FileOrder.Js prior
         /// <param name="provider">The name of the provider responsible for rendering the script output.</param>
         public static void RegisterScript(Page page, string filePath, int priority, string provider)
         {
-            var include = new DnnJsInclude { ForceProvider = provider, Priority = priority, FilePath = filePath};
+            RegisterScript(page, filePath, priority, provider, "", "");
+        }
+
+        /// <summary>
+        /// Requests that a JavaScript file be registered on the client browser
+        /// </summary>
+        /// <param name="page">The current page. Used to get a reference to the client resource loader.</param>
+        /// <param name="filePath">The relative file path to the JavaScript resource.</param>
+        /// <param name="priority">The relative priority in which the file should be loaded.</param>
+        /// <param name="provider">The name of the provider responsible for rendering the script output.</param>
+        /// <param name="name">Name of framework like Bootstrap, Angular, etc</param>
+        /// <param name="version">Version nr of framework</param>
+        public static void RegisterScript(Page page, string filePath, int priority, string provider, string name, string version)
```

Comment:
Is there a reason that `version` would be a `string` instead of a `System.Version` object?